.github/tests/tornjak/pre-install.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+kubectl create namespace spire-system
+kubectl label namespace spire-system pod-security.kubernetes.io/enforce=privileged
+kubectl create namespace spire-server
+kubectl label namespace spire-server pod-security.kubernetes.io/enforce=restricted
+kubectl create namespace tornjak-fe
```

Comment:
Is this split namespace setup required for tornjak. For test simplicity I wouldn't duplicate that test here.